qiime2/sdk/usage.py

```diff
@@ -327,6 +337,13 @@ def _init_metadata_(self, ref, factory):
         })
         return ref
 
+    def _init_data_collection_(self, ref, container_type, *records):
```

Comment:
I think `init_data_collection` is fine.  I don't see any pressing reasons to change it, but I like your previous suggestions better, i.e. `combine_collection_data` or maybe `merge_collection_data`

Reply:
Thanks @andrewsanchez! What about the argument `container_type`? That's what I was asking about above (but I am glad you brought up the function name, too - I think sticking with `init_data_collection` makes sense to me).

Reply:
Oh, I see!  Sorry, totally misunderstood. I think `collection_type` is n improvement since it matches up with the function name.  Happy to go with that!

Reply:
Let's go with `collection_type`, then! Thanks!